includes/transformer/class-base.php

```diff
@@ -162,28 +165,40 @@ public function set_content_visibility( $content_visibility ) {
 	 * @return Base_Object The ActivityPub Object.
 	 */
 	protected function set_audience( $activity_object ) {
-		$public = 'https://www.w3.org/ns/activitystreams#Public';
-		$actor  = Actors::get_by_resource( $this->get_attributed_to() );
-		if ( ! $actor || is_wp_error( $actor ) ) {
-			$followers = null;
-		} else {
+		$public     = 'https://www.w3.org/ns/activitystreams#Public';
+		$followers  = null;
+		$replied_to = null;
+
+		$actor = Actors::get_by_resource( $this->get_attributed_to() );
+		if ( $actor && ! \is_wp_error( $actor ) ) {
```

Comment:
```suggestion
		if ( ! \is_wp_error( $actor ) ) {
```
Is there a chance `Actors::get_by_resource()` returns a falsy value?